lib/libspl/include/sys/isa_defs.h

```diff
@@ -100,28 +100,39 @@ extern "C" {
 
 #define	_SUNOS_VTOC_16
 
-/* arm arch specific defines */
-#elif defined(__arm) || defined(__arm__) || defined(__aarch64__)
+/* aarch64 arch specific defines */
+#elif defined(__aarch64__)
 
 #if !defined(__arm)
```

Comment:
this one is also not needed, right?

Reply:
might need.

Reply:
`__arm` is not defined by compilation environment. But it is referenced at https://github.com/openebs/cstor/blob/develop/lib/libefi/rdwr_efi.c#L156.